components/common-go/baseserver/testing.go

```diff
@@ -34,11 +46,11 @@ func NewForTests(t *testing.T, opts ...Option) *Server {
 	return srv
 }
 
-func MustUseRandomLocalAddress(t *testing.T) *ServerConfiguration {
+func MustUseUniqueLocalAddress(t *testing.T) *ServerConfiguration {
 	t.Helper()
 
 	return &ServerConfiguration{
-		Address: "localhost:0",
```

Comment:
but 0 should pick the available one, no?

Reply:
Yes. Googling hinted at either a) kernel bug or b) misconfigured network stack (tcp4 vs tcp6).
I didn't want to solve here :upside_down_face: